assets/alice.csv

```diff
@@ -906,14 +906,15 @@ ai_alliance_1;The AI is currently seeking allies
 ai_alliance_2;Your nation is close enough to assist in a war
 ai_alliance_3;No conflicting ambitions
 ai_alliance_4;Your nation's military is sufficiently strong
+ai_alliance_5;Has mutual rivals
```

Comment:
Since "rival" is purely an internal name, let's go with something more like "Has shared strategic aims". Also, since this condition works as an OR (as either it or some of the other conditions have be be satisfied) more wording may need to be adjusted